tensorrt_llm/_torch/attention_backend/trtllm.py

```diff
@@ -1277,7 +1321,7 @@ def forward(
             out_scale_sf=out_scale_sf,
             kv_scales_sf=kv_scales_sf,
             kv_scales_sf_inv=kv_scales_sf_inv,
-            use_nvfp4_output=use_nvfp4_output,
+            use_nvfp4_output=output_sf is not None,
```

Comment:
Probably related to my comment above about the multiple "output scaling factors" in the signature. Can we explain how not having an `output_sf` implies `use_nvfp4_output`?